test/test_split.cpp

```diff
@@ -67,67 +67,85 @@ TEST(test_split, split) {
   ASSERT_EQ(RCUTILS_RET_OK, ret);
 
   rcutils_string_array_t tokens1 = test_split("hello_world", '/', 1);
+  ASSERT_EQ(tokens1.size, 1u);
```

Comment:
Are these necessary?  As far as I can tell, the `test_split()` function already does this check based on the final argument.  So I'd suggest we just remove these (unless I'm missing something).

Reply:
Oh, my bad. I missed that when I suggested adding assertions. My apologies @Blast545, but @clalancette is right.